common/flatpak-dir.c

```diff
@@ -277,6 +277,25 @@ get_config_dir_location (void)
   return (const char *) path;
 }
 
+static const char *
+get_run_dir_location (void)
+{
+  static gsize path = 0;
```

Comment:
don't really get why this is `gsize` rather than `char *`

Reply:
That is just how the g_once_xxx APIs work.